sdk/communication/azure-communication-sms/tests/test_sms_client_e2e_async.py

```diff
@@ -106,103 +75,85 @@ async def test_send_sms_from_managed_identity_async(self):
                 from_=self.phone_number,
                 to=[self.phone_number],
                 message="Hello World via SMS")
-            
+
             assert len(sms_responses) == 1
 
             self.verify_successful_sms_response(sms_responses[0])
-    
-    @AsyncCommunicationTestCase.await_prepared_test
+
+    @recorded_by_proxy_async
     async def test_send_sms_fake_from_phone_number_async(self):
+        sms_client = self.create_client_from_connection_string()
 
-        sms_client = SmsClient.from_connection_string(
-            self.connection_str, 
-            http_logging_policy=get_http_logging_policy()
-        )
-        
         with pytest.raises(HttpResponseError) as ex:
             async with sms_client:
                 # calling send() with sms values
                 await sms_client.send(
                     from_="+15550000000",
                     to=[self.phone_number],
                     message="Hello World via SMS")
-        
-        assert str(ex.value.status_code) == "400"
+
+        assert str(
+            ex.value.status_code) == "401"
```

Comment:
The service used to return 400 but the validation changed and it's 401 now. The team is aware of this issue and has a ticket for addressing it. Re-record the test if service is re-deployed with a different error handling logic for this endpoint.